### PR TITLE
Add PHP7.4 support and escape symfony YML config

### PR DESCRIPTION
**Pull request by maxime-rainville**

### CODE REVIEW

File: _config/middleware.yml [hunk 1]
```diff
@@ -7,6 +7,6 @@ SilverStripe\Core\Injector\Injector:
   SilverStripe\Control\Director:
     properties:
       Middlewares:
-        SubsitesStateMiddleware: %$SilverStripe\Subsites\Middleware\InitStateMiddleware
+        SubsitesStateMiddleware: ''%$SilverStripe\Subsites\Middleware\InitStateMiddleware'
```
Comment (dnsl48) [inline _config/middleware.yml]: is this a typo?

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: _config/middleware.yml (hunk 1 above)
Comment (maxime-rainville) [reply _config/middleware.yml]:
```suggestion
        SubsitesStateMiddleware: '%$SilverStripe\Subsites\Middleware\InitStateMiddleware'
```

File: composer.json [hunk 2]
```diff
@@ -23,7 +23,8 @@
         "silverstripe/versioned": "^1.4@dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "sminnee/phpunit": "^5.7.29",
+        "sminnee/phpunit-mock-objects": "^3.4.9",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "sminnee/phpunit": "^5.7.29"
```

File: composer.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
         "silverstripe/versioned": "^1.4@dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "sminnee/phpunit": "^5.7.29"
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "sminnee/phpunit": "^5.7.29",
```

**Comment by dnsl48** [comment]
got another PR, combined with shared travis config - https://github.com/silverstripe/silverstripe-subsites/pull/458

**Pull request closed without merging**